server/image_remove.go

```diff
@@ -17,41 +16,36 @@ func (s *Server) RemoveImage(ctx context.Context, req *pb.RemoveImageRequest) (r
 		recordOperation(operation, time.Now())
 		recordError(operation, err)
 	}()
+	resp = &pb.RemoveImageResponse{}
 
-	logrus.Debugf("RemoveImageRequest: %+v", req)
+	logrus.Debugf("%s: request: %+v", operation, req)
 	image := ""
 	img := req.GetImage()
 	if img != nil {
 		image = img.Image
 	}
 	if image == "" {
-		return nil, fmt.Errorf("no image specified")
+		logrus.Infof("%s: no image specified in request", operation)
+		return resp, nil
```

Comment:
Just to double check: An empty string qualifies as "not parsable", right?

Reply:
Yes :+1: